datalad/support/sshconnector.py

```diff
@@ -460,30 +668,30 @@ class SSHManager(object):
     """
 
     def __init__(self):
+        super().__init__()
         self._socket_dir = None
         self._connections = dict()
         # Initialization of prev_connections is happening during initial
         # handling of socket_dir, so we do not define them here explicitly
         # to an empty list to fail if logic is violated
         self._prev_connections = None
         # and no explicit initialization in the constructor
-        # self.assure_initialized()
+        # self.ensure_initialized()
 
     @property
     def socket_dir(self):
         """Return socket_dir, and if was not defined before,
         and also pick up all previous connections (if any)
         """
-        self.assure_initialized()
+        self.ensure_initialized()
         return self._socket_dir
 
-    def assure_initialized(self):
+    def ensure_initialized(self):
         """Assures that manager is initialized - knows socket_dir, previous connections
         """
         if self._socket_dir is not None:
             return
-        from ..config import ConfigManager
-        cfg = ConfigManager()
+        from datalad import cfg
```

Comment:
just a note to myself: didn't check the logs etc, but may be that original code was on purpose to avoid circular imports? or probably be indeed no longer the case since `ensure_initialized` is commented out in the constructor... (code coverage annotation is silent for me for this PR ATM for some reason :-( )